ci-operator/step-registry/ipi/conf/vsphere/proxy/https/ipi-conf-vsphere-proxy-https-commands.sh

```diff
@@ -3,21 +3,17 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-PROXY_CREDS_PATH=/var/run/vault/vsphere/proxycreds
-ADDITIONAL_CA_PATH=/var/run/vault/vsphere/additional_ca
+proxy_creds=$(< /var/run/vault/vsphere/proxy_creds)
+additional_ca="/var/run/vault/vsphere/client_ca.crt"
 
-proxy_user=$(grep -oP 'user\s*:\s*\K.*' ${PROXY_CREDS_PATH})
-proxy_password=$(grep -oP 'password\s*:\s*\K.*' ${PROXY_CREDS_PATH})
-additional_ca=$(cat ${ADDITIONAL_CA_PATH})
-
-http_proxy_url="http://${proxy_user}:${proxy_password}@172.31.249.80:3128"
-https_proxy_url="https://${proxy_user}:${proxy_password}@172.31.249.80:3130"
+http_proxy_url="http://${proxy_creds}@172.31.249.80:3128"
```

Comment:
How about saving the bastion IP in the vault too.

Reply:
sure, saved ip in the vault, and updated the step.
Failed job is still due to bz 2031049.